src/pages/whats-happening/v11-release/index.mdx

```diff
@@ -0,0 +1,195 @@
+
+### What will the Carbon team be supporting for v10?
+
+- No new functionality will be introduced in v10 after the v11 release.
+
+* We will continue to address high impact bugs that come up for the v10 release
+  after the v11 release.
+
+- We will continue to accept any contributions that look to address issues in
+  the v10 release if we are unable to get to them in time for your product
+  roadmap.
+
+### What is the rule for supporting deprecated things?
+
+- Things that were deprecated from v9 to v10 will be removed in v11.
+
+* Things that are deprecated in v10 will remain in v11 and will be removed in
+  v12.
+
+### Will I need to redesign my product?
+
+- Visually, nothing is changing.
+
+* There are system-side changes that will require changes to design specs and
+  code, like the token names update and inline theming.
+
+- Token names are largely a 1:1 update and very few tokens are keeping their v10
+  name. This means color tokens will have to be updated across assets.
+
+* We are hoping to be able to provide a script that can make these changes on
+  the code side automatically, however this has not been built or tested yet.
+
+### Why rename things instead of leaving things as they are?
+
+- Making these updates now creates a better system user experience and it sets
+  us up for growth and scale so new tokens can fit into the set more easily.
+
+* The new names imply the usage and so will be much easier to apply. For
+  example, `text-03` will be `text-placeholder`.
+
+### How big is the effort to migrate current Sketch designs to the v11 Sketch kit?
+
+- The Carbon Sketch files will automatically update to v11. Connected styles and
+  symbols will automatically receive the v11 updates once the library update is
+  accepted by the users in each design file.
+
+* If you're not ready to update to v11, you can stop accepting library updates
+  to that file until you are ready.
+
+- Sketch files for v10 will be available in the GitHub archive but will no
+  longer be an attached library. There is no way to properly version Sketch
+  Cloud files. There is only the latest version.
+
+* The effort to update local/PAL assets depends on the structure of files. If
+  you are using the Carbon layer styles then the update will be automatic. If
+  you're using hard values and just specing the token names, then you will need
+  to update your specs.
+
+- Read the
+  [Sketch documentation](https://www.sketch.com/docs/libraries/#updating-library-components)
+  for more details.
+
+### How big is the expected effort to migrate code to v11?
+
+- The Carbon team is providing an upgrade experience to automate as much of the
+  migration process as possible.
+
+* A lot of the changes made will be name-based.
+
+- Some changes with components will require updating to the new Component API.
+
+* Teams will need to update their Sass dependency since `node-sass` has been
+  deprecated. They will also need to update how they bring in Carbon.
```

Comment:
The latest medium article has good, detailed information on things teams can do now to prepare for the v11 release, like switching from `node-sass` to `sass` and updating the `bx` prefix. Maybe link to the article in this bullet point for more info on what can be done now?

https://medium.com/carbondesign/carbons-2021-release-april-update-9d23242b3dea